src/main/java/seedu/address/logic/CommandResult.java

```diff
@@ -11,27 +11,43 @@
 
     private final String feedbackToUser;
 
-    /** Help information should be shown to the user. */
+    /**
+     * Help information should be shown to the user.
+     */
     private final boolean showHelp;
 
-    /** The application should exit. */
+    /**
+     * The application should exit.
+     */
     private final boolean exit;
 
+    /**
+     * The application should display candidate list.
+     */
+    private final boolean listC;
+
+    /**
+     * The application should display position list.
+     */
+    private final boolean listP;
+
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, boolean listC, boolean listP) {
```

Comment:
Booleans should be named accordingly. Maybe islistC, islistP?

Reply:
I did this initially to be consistent with the original CommandResult boolean fields naming such as exit and showHelp. Nevertheless, it should be better if I change all. Will make the changes accordingly.